lib/rules/no-unsupported-keywords.js

```diff
@@ -16,12 +16,12 @@ function checkExpression(context, expression, allKeywords) {
   var calleeText = source.getText(expression);
   var assertionText = calleeText.substr(expectText.length + 1);
 
-  var keywords = assertionText.split(/[\.\s]/).filter(function (k) {
+  var keywords = assertionText.split(/[.\s]/).filter(function (k) {
     return !!k;
   });
   for (var i = 0; i < keywords.length; i++) {
     var keyword = keywords[i];
-    if (!allKeywords.hasOwnProperty(keyword)) {
+    if (!{}.hasOwnProperty.call(allKeywords, keyword)) {
```

Comment:
Why this change? Same functionality, the original is slightly more readable IMO. Presume your version has an advantage though?

Reply:
This is part of the rule added to eslint-recommended, at https://eslint.org/docs/rules/no-prototype-builtins . See that page for the rationales.

Reply:
I see. Fine to roll with that, in that case.

Reply:
But actually, it may become a non-issue later if I bump to Node 8 and then refactor this code to just use a for-of loop :)  That ensures it only iterates the "own" properties, so need for the extra check anyways.